src/main.jsx

```diff
@@ -79,8 +79,18 @@ const setupApp = async persistedState => {
         Alerter.info('Token refreshed')
       }
     })
-
+    const openUniversalLink = (/*eventData*/) => {
+      /* 
+      banks just need to be waked up by an universal link. 
+      We currently do not manage any path
+    */
+    }
```

Comment:
The only place where banks need to resolve UL is during the onboarding process. It'll be done in cozy-auth

Reply:
So this is not necessary ?

Reply:
"ok laisse le" :D